Mathlib/Analysis/InnerProductSpace/Symmetric.lean

```diff
@@ -74,17 +74,48 @@ theorem IsSymmetric.apply_clm {T : E →L[𝕜] E} (hT : IsSymmetric (T : E →
     ⟪T x, y⟫ = ⟪x, T y⟫ :=
   hT x y
 
+@[aesop safe apply]
 theorem isSymmetric_zero : (0 : E →ₗ[𝕜] E).IsSymmetric := fun x y =>
```

Comment:
The name change will require a deprecation for the old name, and it will like cause some fallout which will need fixing.
```suggestion
@[simp]
protected theorem IsSymmetric.zero : (0 : E →ₗ[𝕜] E).IsSymmetric := fun x y =>
```

Reply:
The way you have written this suggests that I need to declare a deprecation for the old name. Do I need to do anything other than change the name (and simp attribute)?

Reply:
Yes, you need to add
```
@[deprecated (since := "2024-09-30")] alias isSymmetric_zero := IsSymmetric.zero
```